docs/topics/modules/api_get_component_analyses_response.adoc

```diff
@@ -36,75 +34,68 @@ curl -k -H "Authorization: Bearer <openshift.io authentication token>" https://<
 [source,typescript]
 ----
 {
+    "schema": {
+        "url": "http://recommender.api.openshift.io/api/v1/schemas/api/analyses_graphdb/1-2-0/",
```

Comment:
do we need to expose http://recommender.api.openshift.io as we asked for 3scale gaeway url

Reply:
@samuzzal-choudhury can you please confirm if this needs to be replaced and if so with what?

Reply:
@Preeticp  Let's replace `http://recommender.api.openshift.io` with `http://<recommender api url>`.